test/scripts/e2e.sh

```diff
@@ -164,26 +164,47 @@ if [ -z "$E2E_TEST_FILTER" ] || [ "$E2E_TEST_FILTER" == "SCRIPTS" ]; then
 
     # If the platform is arm64, we want to pass "--keep-temps" into e2e_client_runner.py
     # so that we can keep the temporary test artifact for use in the indexer e2e tests.
-    # The file is located at ${TEMPDIR}/net_done.tar.bz2
-    if [ "$E2E_PLATFORM" == "arm64" ]; then
+    # The file is located at ${TEMPDIR}/${AWS_NET_UPLOAD}
+    # TODO: Do I still need FORCE_KEEP_TEMPS ?
+    if [ "$E2E_PLATFORM" == "arm64" ] || [ -n "$FORCE_KEEP_TEMPS" ]; then
       KEEP_TEMPS_CMD_STR="--keep-temps"
     fi
 
     "${TEMPDIR}/ve/bin/python3" e2e_client_runner.py ${KEEP_TEMPS_CMD_STR} ${RUN_KMD_WITH_UNSAFE_SCRYPT} "$SRCROOT"/test/scripts/e2e_subs/*.{sh,py}
 
     # If the temporary artifact directory exists, then the test artifact needs to be created
     if [ -d "${TEMPDIR}/net" ]; then
+            if [ -z "$AWS_NET_UPLOAD" ]; then
+            AWS_NET_UPLOAD="net_done.tar.bz2"
+        fi
+
         pushd "${TEMPDIR}" || exit 1
-        tar -j -c -f net_done.tar.bz2 --exclude node.log --exclude agreement.cdv net
+        tar -j -c -f ${AWS_NET_UPLOAD} --exclude node.log --exclude agreement.cdv net
         rm -rf "${TEMPDIR}/net"
         RSTAMP=$(TZ=UTC python -c 'import time; print("{:08x}".format(0xffffffff - int(time.time() - time.mktime((2020,1,1,0,0,0,-1,-1,-1)))))')
-        echo aws s3 cp --acl public-read "${TEMPDIR}/net_done.tar.bz2" s3://algorand-testdata/indexer/e2e4/"${RSTAMP}"/net_done.tar.bz2
-        aws s3 cp --acl public-read "${TEMPDIR}/net_done.tar.bz2" s3://algorand-testdata/indexer/e2e4/"${RSTAMP}"/net_done.tar.bz2
+        echo aws s3 cp --acl public-read "${TEMPDIR}/${AWS_NET_UPLOAD}" s3://algorand-testdata/indexer/e2e4/"${RSTAMP}"/${AWS_NET_UPLOAD}
+        aws s3 cp --acl public-read "${TEMPDIR}/${AWS_NET_UPLOAD}" s3://algorand-testdata/indexer/e2e4/"${RSTAMP}"/${AWS_NET_UPLOAD}
         popd
     fi
 
     duration "parallel client runner"
 
+
+    echo "CHANNEL = $CHANNEL"
+    echo "BINDIR = $BINDIR"
+    echo "DATADIR = $DATADIR"
+    echo "E2E_TEST_FILTER = $E2E_TEST_FILTER"
+    echo "E2E_PLATFORM = $E2E_PLATFORM"
+    echo "INTERACTIVE = $INTERACTIVE"
+    echo "TEMPDIR = $TEMPDIR"
+    echo "FORCE_KEEP_TEMPS = $FORCE_KEEP_TEMPS"
+    echo "KEEP_TEMPS_CMD_STR = $KEEP_TEMPS_CMD_STR"
+    echo "E2E_TEST_FILTER = $E2E_TEST_FILTER"
+    echo "AWS_NET_UPLOAD = $AWS_NET_UPLOAD"
+
+    exit 42
```

Comment:
Seems excessive?

42?

Reply:
all experimental and should have been in a seperate branch.